testsuite/forge-cli/src/main.rs

```diff
@@ -1952,9 +1953,9 @@ fn realistic_env_max_load_test(
         .add_system_metrics_threshold(SystemMetricsThreshold::new(
             // Check that we don't use more than 18 CPU cores for 10% of the time.
             MetricsThreshold::new(18.0, 10),
-            // Memory starts around 3GB, and grows around 1.2GB/hr in this test.
-            // Check that we don't use more than final expected memory for more than 10% of the time.
-            MetricsThreshold::new_gb(3.3 + 1.4 * (duration_secs as f64 / 3600.0), 10),
+            // Memory starts around 3.5GB, and grows around 1.5GB/hr in this test.
+            // Check that we don't use more than final expected memory for more than 20% of the time.
+            MetricsThreshold::new_gb(3.5 + 1.5 * (duration_secs as f64 / 3600.0), 20),
```

Comment:
Is this change necessary after #13906? I'm thinking it might just be a side effect of having increased the block sizes inadvertently.

Reply:
You are right. If the other PR lands first, this may not be required.